ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs

```diff
@@ -15,13 +15,13 @@
   </page.header>
 
   <page.actions>
-    {{#if @model}}
-      <Rose::LinkButton
-        @route='scopes.scope.host-catalogs.new'
-        @style='primary'
-      >
-        {{t 'titles.new'}}
-      </Rose::LinkButton>
+    {{#if (can 'create collection' this.scope collection='host-catalogs')}}
+      {{#if @model}}
+        <Rose::LinkButton
+          @route='scopes.scope.host-catalogs.new'
+          @style='primary'
+        >{{t 'titles.new'}}</Rose::LinkButton>
```

Comment:
The formatting here seems odd? can we move `</Rose:LinkButton>` to the next line and make it match with the indentation when we open the tag?

Reply:
good catch. will run format on this!